cpp/oneapi/dal/algo/decision_forest/backend/gpu/train_kernel_cls_hist_dpc.cpp

```diff
@@ -32,15 +32,20 @@ template <typename Float>
 static result_t call_daal_kernel(const context_gpu& ctx,
```

Comment:
Its better to name it `train_hist` or `train_kernel_hist`, because `call_daal_kernel` is not representative